### PR TITLE
apply coding rules

### PR DESCRIPTION
**Pull request by Howard-C**
to `rendermidi.cpp`

### CODE REVIEW

File: libmscore/rendermidi.cpp [hunk 1]
```diff
@@ -1584,24 +1584,24 @@ void renderArpeggio(Chord *chord, QList<NoteEventList> & ell)
...
-int convertLine (int lineL2, ClefType clefL, ClefType clefR) {
+int convertLine(int lineL2, ClefType clefL, ClefType clefR) {
       int lineR2 = lineL2;
       int goalpitch = line2pitch(lineL2, clefL, Key::C);
       int p;
-      while ( (p = line2pitch(lineR2, clefR, Key::C)) > goalpitch && p < 127)
+      while ((p = line2pitch(lineR2, clefR, Key::C)) > goalpitch && p < 127)
             lineR2++;
-      while ( (p = line2pitch(lineR2, clefR, Key::C)) < goalpitch &&  p > 0)
+      while ( p = line2pitch(lineR2, clefR, Key::C)) < goalpitch && p > 0)
```
Comment (Jojo-Schmitz) [inline libmscore/rendermidi.cpp]: Syntax error

Comment (Howard-C) [reply libmscore/rendermidi.cpp]: Yes it is!

**Comment by dmitrio95** [comment]
Unfortunately this is likely to cause a lot of conflicts with #5264, would you mind postponing this until merging #5264?

**Comment by Howard-C** [comment]
I wouldn't "mind". I'm actually looking for something else but these code inconsistencies disturb me so I decided to do this.

**Pull request closed without merging**